src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -671,7 +738,7 @@ private static void StopOrleansSilo(SiloHandle instance, bool stopGracefully)
             catch (Exception exc)
             {
                 Console.WriteLine(exc);
-                throw;
+                //throw;
```

Comment:
Might want to add to the WriteLine above something along the lines of "While stopping silo {instance} ignoring error {exception}" to make it clearer [a] what that console message relates to, [b] that the error is being treated as non-fatal, and [c] which silo instance caused the problem.